detail.html

```diff
@@ -0,0 +1,35 @@
+<%inherit file="app_base.htm" />
+
+<%block name="content_middle">
+  <h1 class="myheader">${ product_name }</h1>
+
+  <div id="mycontainer">
+
+    <div id="image_sidebar">
+      <ul>
+        %for image in product.image_urls():
+        <li class="">
+          <img class="mythumbnail" alt="an alternate image for ${ product_name }" src="${ image }" />
+        </li>
+        %endfor
+      </ul>
+    </div>
+
+    <div id="image_viewer">
+      <img id="main_image" alt="main picture for ${ product_name }" src="${ product.image_url() }" />
```

Comment:
I would recommend including the price on the detail page underneath the product name.

Reply:
That would be very useful. I guess I was thinking more in terms of project requirements instead of what's most effective! 😢